.golangci.yml

```diff
@@ -10,36 +11,35 @@ linters:
     # - funlen
     # - gochecknoglobals
     # - gochecknoinits
+    # - gocognit
     - goconst
-    - gocritic
+    # - gocritic
     # - gocyclo
     # - godox
     - gofmt
     - goimports
-    - golint
```

Comment:
why remove?

Reply:
this was pulled from upstream, and and while I couldn't find an explicit comment on why, I believe it is caused by this warning after updating to 1.42.0 go golangci-lint

```
WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive.
```